themes/custom/droopler_theme/js/global.js

```diff
@@ -58,7 +58,22 @@
         $(this).parent().removeClass('force-show');
         $(this).parent().find(".dropdown-menu").removeClass('force-show');
       } });
-      $body.addClass("d-theme-preceded")
+      $body.addClass("d-theme-preceded");
+
+      // Check if alert box is visible on hanging-header and adjust position.
```

Comment:
Może by to wszystko wrzucić w if i uzależnić od tego, że header with cta jest wybrany jako layout node'a? IMO byłoby czytelniej. Jeśli mamy klasyczny layout to nie musimy nic robić z alertami.